lib/fuse_loop_mt.c

```diff
@@ -44,6 +45,17 @@ struct fuse_mt {
 	int error;
 };
 
+struct fuse_in_header {
```

Comment:
do not copy the struct. there should be another way to solve the conflict of the types